lib/fly.js

```diff
@@ -85,12 +85,18 @@ module.exports = Fly
  * @return                      The current Fly instance.
  */
 Fly.prototype.source = function (globs) {
-	globs = arrify(globs)
+	globs = flatten([].slice.call(arguments))
+	var options = globs.pop()
+	if (typeof options !== 'object') {
+		globs.push(options)
+		options = {}
+	}
```

Comment:
`source` *would be* a function of `globs` and `options`. If no options, then default to `{}`

```js
source = function (globs, source) {
  globs = arrify(globs);
  options = options || {};
  // ...
}
```